<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KMU-Beispiel-basierte Erklärung von Anomalieerkennung in Ventilen durch Audioklassifikation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFileBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auswählen...</string>
    </property>
   </widget>
   <widget class="QLabel" name="audioDataLbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Audiodatei auswählen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nearMissLbl">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Near misses:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showNearMissBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzeigen</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nearMissLimitBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="nearMissAmountLbl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Anzahl:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAndClassifyBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Spektrogramm Anzeigen und Klassifizieren</string>
    </property>
   </widget>
   <widget class="QListWidget" name="audioFileList">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
border: 1px solid black;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playAudioBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>abspielen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>pausieren</string>
    </property>
   </widget>
   <widget class="QLabel" name="WaitLblNearMiss">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777209</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Bitte Warten ...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
